linux-armv5/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM dockcross/base:latest
-MAINTAINER Matt McCormick "matt.mccormick@kitware.com"
+FROM dockcross/base
+MAINTAINER Roman Valls Guimera "brainstorm@nopcode.org"
```

Comment:
Please leave these two unrelated lines unchanged.

Reply:
You want to be the maintainer of the xtensa support?

Reply:
It would be great if you could be the maintainer for `xtensa`.

However, this is changing `linux-armv5`.

Also, the `:latest` tag should be used in FROM per Dockerfile best practices.

Reply:
Ouch, that was embarassing, sorry about that, will fix it ;)